src/robot_arm.hpp

```diff
@@ -33,17 +32,38 @@ class RobotArm {
      *
      */
     RobotArm();
+    /**
+     * @brief intToChar function
+     *
+     * This function takes an integer number and converts it to a char *. The char * will be stored in the 'p' parameter
+     *
+     * @param i : int
+     * @param p  : char *
```

Comment:
You should explain your varibles here. I can see the type in the function so it's more important to give some explaination.